justfile

```diff
@@ -3,6 +3,28 @@ default:
     @just --list
 
 
+# Run all autoformattings
+[no-exit-message]
+format:
+    ruff format .
+    yamlfmt .
+
+
+# Run all linters
+[no-exit-message]
+lint:
+    #!/usr/bin/env -S parallel --shebang --ungroup --jobs {{ num_cpus() }}
```

Comment:
This parallelizes the below commands. The [just docs](https://just.systems/man/en/running-tasks-in-parallel.html?highlight=parallel#running-tasks-in-parallel) recommend to use the lesser-known [GNU parallel](https://savannah.gnu.org/projects/parallel/) over the more portable xargs. Probably because xargs would interweave the outputs of the different commands.

Therefore, make sure to install `brew install parallel`